.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "python.linting.flake8Enabled": true,
+    "python.linting.enabled": true,
+    "python.formatting.provider": "yapf"
+}
```

Comment:
Please add .vscode/ to the .gitignore these settings should not be in the main repository

Reply:
surely it makes sense to include things like this in the repo, would help enforce consistency between our code?

Reply:
Not everyone uses VsCode, it just pollutes the repo and will not be needed when the repo gets deployed to a server